README.md

```diff
@@ -1,114 +1,235 @@
-IBM Bluemix Mobile Services - Client SDK Android Core
+BMSCore
 ===
 
 [![Build Status](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core.svg?branch=master)](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core)
-[![Build Status](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core.svg?branch=development)](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.ibm.mobilefirstplatform.clientsdk.android/core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.ibm.mobilefirstplatform.clientsdk.android/core)
```

Comment:
Are you referring to the Build Status, or Maven Central badge? The Maven Central badge is still there, just moved to a different spot. I think it's confusing to have 2 badges for Build Status, especially when one is passing and one is failing. Most Github repos have just one build status.